.gitignore

```diff
@@ -24,5 +24,4 @@ yarn-error.log*
 
 /.vscode
 /.eslintcache
-.idea
-/.husky/
+.idea
```

Comment:
Why not this?
```suggestion
.idea
/.husky/**/**.*
```

Reply:
Because if I do it like that git wants to add the folder content into de project.

Reply:
But with your changes, I remove the .husky folder, then do a `yarn install` and when I try to commit, husky files and folder appear. 
![image](https://user-images.githubusercontent.com/2439363/132052656-105e20bc-073b-496a-a847-15024f4353e3.png)